doc/hcr.rst

```diff
@@ -187,7 +189,9 @@ Native code targets
 Native projects using the hot code reloading option will be implicitly
 compiled with the `-d:useNimRtl` option and they will depend on both
 the ``nimrtl`` library and the ``nimhcr`` library which implements the
-hot code reloading run-time.
+hot code reloading run-time. Both libraries can be found in the ``lib``
+folder of Nim and can be compiled to a ``.dylib`` file to satisfy compiling
+of the example code above.
```

Comment:
I think this should describe how to compile them instead.

Reply:
I've added some example console commands on UNIX/MacOS

I was not sure whether to use RST `::` or use `.. code-block:: console` (used the former as it's how it's written in this document)

commit `daccd8464b4186013ef5b59d13b3939512e2fdc1` is a suggestion on this.